code/lib/builder-manager/templates/template.ejs

```diff
@@ -2,13 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta name="theme-color" content="<%= themeColor %>" />
```

Comment:
@tmeasday Do you have any ideas on how to get the theme color to show here? If not, I can pull this line of code.

Reply:
Hmmm, not sure, I think @ndelangen knows the most about themes

Reply:
No way to do it there, as the theme is determined at runtime.
And in fact can change at runtime.

At least that's how it is at this point, and will likely stay in 7.0 because changing that will be a breaking change.

Currently users define their theme in `manager.ts` but in order to know it at build time, it'd have to be moved to `main.ts` or some preset value.

Reply:
Thanks for the heads up Norbert! @MichaelArestad let's punt on this meta tag